### DISCUSSION TITLE
Bug: We are having problems searching workflow runs. The results may not be complete.

### DISCUSSION THREAD

**Discussion by cbeauchesne** [community/community#14871] [Actions]
When I filter actions using a branch, some results are missing, as explained by the red banner that appears at the top of the page:

> We are having problems searching workflow runs. The results may not be complete.

This bug also impacts actions API, and it breaks a lot of our CIs 

Where can I follow this bug to know when it’ll be resolved ?

Cheers !

**Comment by gustavo-salazar** [comment]
Seems we are having the same issue. 

I found this answer in the community forum: https://github.community/t/filtering-workflow-runs-does-not-include-runs-created-in-the-last-6-hours/244282/4?u=gustavo-salazar
